src/style.css

```
@@ -1 +1,62 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", serif, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 520;
```

there are only 400. 500. 600 etc.
```suggestion
  font-weight: 500;
```